Sources/Internal/LayoutItem.swift

```diff
@@ -66,13 +70,26 @@ extension LayoutItem.ItemType: Comparable {
       return lhsMonth < rhsMonth
     case let (.dayOfWeekInMonth(_, lhsMonth), .day(rhsDay)):
       return lhsMonth <= rhsDay.month
+    case let (.dayOfWeekInMonth(_, lhsMonth), .monthFooter(rhsMonth)):
+      return lhsMonth < rhsMonth
```

Comment:
```suggestion
      return lhsMonth <= rhsMonth
```

Same thing here; The day of the week items are at the beginning of the month, so even for the same month, these come "before" the footer, which is the last item.